ITEM_en.tsv

```diff
@@ -4660,9 +4660,9 @@ ITEM_20150428_005046	It seems like it was left here as a last ditch effort by so
 ITEM_20150428_005047	A part of a pilgrim's torn diary. Readable.
 ITEM_20150428_005048	A hastily scribbled record of a pilgrim. Readable.
 ITEM_20150428_005049	A will left by a dying pilgrim. Readable.
-ITEM_20150428_005050	A burnt journal left here by a pilgrim. Readable.
+ITEM_20150428_005050	Burnt journal left by pilgrims. It's readable.
 ITEM_20150428_005051	A lost diary of some kind. Readable.
-ITEM_20150428_005052	A journal that seems to have been written by someone who was angry. Seems readable upon closer examination.
+ITEM_20150428_005052	Seems like a scribbled diary written by someone in rage. Looking carefully, it seems readable.
```

Comment:
Incorrect, the item does not "seem to be a diary", it IS a journal that "seems to have been written by someone who as angry." The implications are different.